src/proxy-ui-api/frontend/src/store/modules/addClient.ts

```diff
@@ -112,12 +112,33 @@ export const actions: ActionTree<AddClientState, RootState> = {
       });
   },
 
+  fetchSelectableForSubsystem({ commit, rootGetters }, client: Client) {
+    // Fetch clients from backend that match the selected client without subsystem code
+    return api.get(`/clients?instance=${client.instance_id}&member_class=${client.member_class}&member_code=${client.member_code}&internal_search=false&show_members=false&is_not_local_client=true`)
+      .then((res) => {
+        commit('storeReservedClients', res.data);
+      })
+      .catch((error) => {
+        throw error;
+      });
+  },
+
+  fetchReservedSubsystems({ commit, rootGetters }, client: Client) {
+    // Fetch clients from backend that match the selected client without subsystem code
+    return api.get(`/clients?instance=${client.instance_id}&member_class=${client.member_class}&member_code=${client.member_code}&internal_search=true`)
```

Comment:
Same as above

Reply:
Here the internal_search is using true which I suppose is the right thing here. Now I moved the whole method out of vuex, but the call is still same.